explorer/cnquery_explorer.proto

```diff
@@ -355,7 +355,8 @@ message UpdateAssetJobsReq {
 // Store results for a given asset
 message StoreResultsReq {
   string asset_mrn = 1;
-  map<string, cnquery.llx.Result> data = 3;
+  map<string, cnquery.llx.Result> query_data = 3;
```

Comment:
While there is no technical reason, I'd like to do avoid the rename for now. Can we keep the old name for the field and just extend the comments?

Reply:
yes, happy to adjust it